workflows/n_t.snake

```diff
@@ -185,268 +229,199 @@ rule sp_download:
         cd ../
         """
 
+
 rule run_stairwayplot:
     input:
-        output_dir + "/simulated_data/{demog}/{dfes}/{annots}/{seeds}/sim_{chrms}.trees",
         rules.sp_download.output,
-    output: output_dir + "/inference/sp/{demog}/{dfes}/{annots}/{seeds}/{chrms}/stairwayplot_estimated_Ne.txt"
+        expand(output_dir + "/simulated_data/{{demog}}/{{dfes}}/{{annots}}/{{seeds}}/sim_{chrms}.trees",
```

Comment:
are the double curlies required here (and other places)?

Reply:
may not need them in an expand, probably need them in `shell` rule

Reply:
Yes, they are necessary. Esp as this is working atm I would rather not start tinkering with the expand statements ...